modules/c-blosc2/2.12.0/presubmit.yml

```diff
@@ -3,9 +3,11 @@ matrix:
   - ubuntu2004
   - macos
   - windows
+  bazel: [6.x, 7.x] 
```

Comment:
Modules are "add only" - do not fix anything in 2.12.0 folder - see https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/bcr-policies.md **"Add-only: existing module versions must not be modified to ensure reproducible builds."**